driver/generic/ili9xxx.py

```diff
@@ -77,9 +86,18 @@
     const(_MADCTL_MX | _MADCTL_MY | _MADCTL_MV),  # 3 = inverted landscape
 )
 
+ILI9XXX_PORTRAIT = const(0)
```

Comment:
For completeness and clarity, since your driver is based on st77xx anyway, why not set it like this? :
```python
ILI9XXX_PORTRAIT = st77xx.ST77XX_PORTRAIT
...
```

Reply:
addressed